ui/src/layout/Layout.js

```diff
@@ -13,7 +14,13 @@ const useStyles = makeStyles({
 })
 
 export default (props) => {
-  const theme = useSelector((state) => themes[state.theme] || themes.DarkTheme)
+  const prefersLightMode = useMediaQuery('(prefers-color-scheme: light)')
+  const theme = useSelector((state) => {
+    if (state.theme === 'AUTO') {
+      return prefersLightMode ? themes.LightTheme : themes.DarkTheme
+    }
+    return themes[state.theme] || themes.DarkTheme
+  })
```

Comment:
Does this piece of code do the same as the one above in the `Player.js`? If yes, you could create a hook, (`useCurrentTheme()`?) encapsulating it to avoid duplication.

Reply:
Done. Added the custom hook [here](https://github.com/navidrome/navidrome/pull/835/files#diff-575c4b01cc1430dac2bc035c2c4c150004554bde2e87c782a4bce165411ad7feR17).